src/main/groovy/nebula/plugin/dependencylock/DependencyLockReader.groovy

```diff
@@ -35,8 +34,8 @@ class DependencyLockReader {
 
         if (updates) {
             locks = locks.collectEntries { configurationName, deps ->
-                if (configurationName != conf.name) {
-                    // short-circuit if this is not the relevant configuration
+                if (configurationName != conf.name && configurationName != "_global_") {
```

Comment:
I see there is a static variable imported which has a global config name so you don't need to use it explicitly.

Reply:
`DependencyLockTaskConfigurer.GLOBAL_LOCK_CONFIG`

Reply:
Indeed, thanks!